lib/guest_installation_and_configuration_base.pm

```diff
@@ -1569,6 +1592,24 @@ sub do_detach_guest_installation_screen {
     return $self;
 }
 
+#Return true if guest has [guest_autoconsole] and [guest_noautoconsole] that are not equal to 'none', 'true' or empty which indicates guest definitely has autoconsole.
+#Empty value may indicate there is autoconsole or the opposite which depends detailed configuration of guest.
+sub has_autoconsole_for_sure {
+    my $self = shift;
+
+    $self->reveal_myself;
+    return (($self->{guest_autoconsole} ne 'none') and ($self->{guest_autoconsole} ne '') and ($self->{guest_noautoconsole} ne 'true') and ($self->{guest_noautoconsole} ne ''));
+}
+
+#Return true if guest has [guest_autoconsole] or [guest_noautoconsole] that are equal to 'none' or 'true' which indicates guest definitely has no autoconsole.
+#Empty value may indicate there is autoconsole or the opposite which depends detailed configuration of guest.
+sub has_noautoconsole_for_sure {
+    my $self = shift;
+
+    $self->reveal_myself;
+    return (($self->{guest_autoconsole} eq 'none') or ($self->{guest_noautoconsole} eq 'true'));
+}
+
```

Comment:
Finally, a short reminder -- some lines' length is a little bit too long. Better to pay attention.

Reply:
To be honest, quite some lines, if you scroll up and down to overview the files -- many lines are beyond screen horizontally.

Reply:
For small screen or lower resolution, this might not be avoidable. Actually I already trimmed some lines before submitting this pull request. So if you do not point them out very explicitly, I recommend do not change them because there is no standard about such thing and sometimes 'a long line' just happens.

Reply:
We can have the long lines as they are now. I won't give you any stric precise length to follow. But I hope you can make a quick google search about classic programming recommendations for the  maximum length per line.  It is not good coding habbit to have too~ long lines.